src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/_params.py

```diff
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import (
+    get_resource_name_completion_list,
+    enum_choice_list,
+    name_type,
+    ignore_type)
+from azure.cli.core.commands import register_cli_argument
+import azure.cli.core.commands.arm # pylint: disable=unused-import
+from azure.cli.command_modules.documentdb.sdk.models.document_db_enums import KeyKind
+from azure.cli.command_modules.documentdb.sdk.models.document_db_enums import DefaultConsistencyLevel
+from azure.cli.command_modules.documentdb.sdk.models.failover_policy import FailoverPolicy
+from azure.cli.command_modules.documentdb.sdk.models.location import Location
+
+
+def validate_failover_policies(ns):
+    ''' Extracts multiple space-separated failoverPolcies in region=failoverPriority format '''
+    fp_dict = []
+    for item in ns.failover_policies:
+        comps = item.split('=', 1)
+        fp_dict.append(FailoverPolicy(comps[0], int(comps[1])))
+    ns.failover_policies = fp_dict
+
+def validate_locations(ns):
```

Comment:
since these are identical in logic, you could just have "validate_failover_policies_or_locations" or something. If in the future you think they might be handled differently, then leaving them like this is fine. If they will always move together, then this becomes an added maintenance burden for your team.